scripts/test

```diff
@@ -14,4 +14,11 @@ pep8 --max-line-length=1000 "${GIT_ROOT}"/cnxeasybake/tests
 
 coverage run --source=cnxeasybake setup.py test
 
+if [ "${CI}" = "true" ]
```

Comment:
this doesn't look like an evaluation. should `=` be `==`?

Reply:
also is `;` needed here?

Reply:
We talked about this on slack but I wanted to include it here as well:

> this doesn't look like an evaluation. should `=` be `==`?

This is `/bin/sh` instead of `/bin/bash` so that appears to be the way to do it...  I tested it by doing:

```
karen@yachiyo:~$ /bin/sh -c 'CI=true; if [ "${CI}" = "true" ]; then echo yes; fi'
yes
karen@yachiyo:~$ /bin/sh -c 'CI=true; if [ "${CI}" == "true" ]; then echo yes; fi'
/bin/sh: 1: [: true: unexpected operator
```

> also is `;` needed here?

A newline can be used instead of `;` so you can either do:

```
if [ "${CI}" = "true" ]; then
```

or

```
if [ "${CI}" = "true" ]
then
```